# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1252498472 -7200
#      Wed Sep 09 14:14:32 2009 +0200
# Branch release-3-2-x
# Node ID 22395d0870fdf5f0fd37328086d4352be10db2c7
# Parent  6ba99f799ec1ea3f2c413e38df2b2ed16b05df65
dlmwrite.m: fix typo

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2009-09-08  John W. Eaton  <jwe@octave.org>
+
+	* io/dlmwrite.m: Fix typo.
+
 2009-09-05  John W. Eaton  <jwe@octave.org>
 
 	* plot/hold.m: Correctly toggle hold state.  Set both figure and
 	axes "nextplot" property when turning hold state on.
 	* plot/ishold.m: Check figure and axes nextplot properties.
 	Accept axes or figure handle argument.
 
 2009-08-17  Jaroslav Hajek  <highegg@gmail.com>
diff --git a/scripts/io/dlmwrite.m b/scripts/io/dlmwrite.m
--- a/scripts/io/dlmwrite.m
+++ b/scripts/io/dlmwrite.m
@@ -80,17 +80,17 @@
 ## * Initial revision
 ## 2005-11-27 Bill Denney <bill@givebillmoney.com>
 ## * Significant modifications of the input arguements for additional
 ## functionality.
 
 function dlmwrite (file, a, varargin)
 
   if (nargin < 2 || ! ischar (file))
-    ptint_usage ();
+    print_usage ();
   endif
 
   ## set defaults
   delim = ",";
   r = 0;
   c = 0;
   newline = "\n";
   if (ischar (a))
